utils/make_changelog.py

```diff
@@ -262,7 +262,7 @@ def process_unknown_commits(commits, commits_info, users):
 # Returns False if the PR should not be mentioned changelog.
 def parse_one_pull_request(item):
     description = item['description']
-    lines = [line for line in [x.strip() for x in description.split('\n') if description else []] if line]
+    lines = [line for line in [x.strip() for x in description.split('\n') if description] if line]
```

Comment:
This does not looks correct:

```python
def parse(description): return [line for line in [x.strip() for x in description.split('\n') if description] if line]
parse(False)
AttributeError: 'bool' object has no attribute 'split'
```

I guess this one is correct:

```suggestion
    lines = [line for line in [x.strip() for x in description.split('\n')] if line] if description else []
```

Also map-like approach looks better:

```suggestion
    lines = description.split('\n') if description else []
    lines = map(lambda l: l.strip(), lines)
    lines = map(lambda l: re.sub(r'\s+', ' ', l), lines)
    lines = [*lines]
```

P.S. And does description really can be omitted? In this case it will fail earlier with `KeyError`

Reply:
I didn't assess the logic - I just made a quick-fix looking at your changes to this line.